extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -859,6 +859,12 @@ tags: {  # <amp-story-page-attachment> with href
     value: "dark"
     value: "light"
   }
+  attrs: {
+    name: "cta-text"
+  }
+  attrs: {
```

Comment:
Is it worth to make `title` and `data-title` exclusive (and same thing with `cta-text` and `data-cta-text`)?

Reply:
Doesn't look like that's an option, without making one of them mandatory. The code overrides data- attributes when title or cta-text are set as well